src/overmap.cpp

```diff
@@ -2668,13 +2698,19 @@ tripoint overmap::draw_overmap(const tripoint &orig, const draw_data_t &data)
                 tmp.y = locations[i].y;
                 draw(g->w_overmap, g->w_omlegend, tmp, orig, uistate.overmap_show_overlays, show_explored, NULL, draw_data_t());
                 //Draw search box
-                draw_border(w_search);
-                mvwprintz(w_search, 1, 1, c_red, _("Find place:"));
+                mvwprintz(w_search, 1, 1, c_red, _("Find place: %*d/%d"), 6, i+1, locations.size());
```

Comment:
Might recommend changing this to "Search results" now that there's a number sitting next to the string

<img width="228" alt="screen shot 2017-08-29 at 1 16 01 pm" src="https://user-images.githubusercontent.com/714017/29834227-3416d6e8-8cbc-11e7-89c3-4aa21abe7295.png">